crates/server/README.md

```diff
@@ -0,0 +1,291 @@
+# Node Server
+
+- [Introduction](#introduction)
+    - [Admin API](#1admin-api)
+    - [JSON rpc](#2json-rpc)
+    - [Websocket](#3websocket)
+- [Node Server Workflows](#node-server-workflows)
+    - [Client Login Workflow](#client-login-workflow)
+    - [JSON rpc Workflow](#json-rpc-workflow)
+    - [Websocket Workflow](#websocket-workflow)
+- [Admin API endpoints](#admin-api-endpoints)
+    - [Protected Routes](#protected-routes)
+    - [Unprotected Routes](#unprotected-routes)
+- [JSON rpc endpoint](#json-rpc-endpoint)
+- [Websocket endpoints](#websocket-endpoints)
+- [Examples](#examples)
+
+## Introduction
+
+Node Server is a component in node that facilitates node administration and enables communication with the logic of an application (loaded wasm) in participating contexts.
+
+Node Server component is split into 3 parts:
+ 1. [Admin API](https://github.com/calimero-network/core/blob/feat-admin_api_docs/crates/server/src/admin/service.rs)
+ 2. [JSON rpc](https://github.com/calimero-network/core/blob/feat-admin_api_docs/crates/server/src/jsonrpc.rs)
```

Comment:
Also, can we fix the casing of "RPC"

In some places, it's "rpc" in other's it's "RPC"

Reply:
renamed to "rpc"